style.css

```diff
@@ -95,3 +93,279 @@ h2 {
 .social-link-item {
   margin-right: 12px;
 }
+
+.react-work {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  height: fit-content;
+}
+
+.button1 {
+  height: 48px;
+  width: 124px;
+  margin-left: 24px;
+  color: white;
+  background-color: #ff6b00;
+  border-radius: 0;
+  border: none;
+}
+
+.title-text {
+  height: 107.00003051757812px;
+  width: 327px;
+  left: 24px;
+  top: 120px;
+  border-radius: 0;
+  margin-top: 120px;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.work {
+  height: 237px;
+  width: 327px;
+  left: 24px;
+  top: 0;
+  border-radius: 0;
+  margin-top: 120px;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.multi {
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-top: 35px;
+}
+
+.pharagraph {
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-top: 20px;
+}
+
+.work-choise {
+  display: grid;
+  grid-template-columns: 15% 15% 28% 15%;
+  list-style-type: none;
+  gap: 19px;
+  padding: 24px;
+  padding-right: 5px;
+  padding-top: 6px;
+  padding-bottom: 0;
+}
+
```

Comment:
## Changes Required
⚠️ According to project guidelines: 
For each project, the content of the project is aligned with the flexbox.  You did a great job using grid! ✨ But it's supposed to be used for the parent `works` section, not for every project. Please, kindly fix this.